client/tests/component/support/mocks.ts

```diff
@@ -32,6 +32,7 @@ function getDefaultProvideConfig(timeSince = mockTimeSince, fileSize = mockFileS
 }
 
 function mockI18n(): void {
+  // import { useI18n } from 'vue-i18n';
```

Comment:
if useless remove comment?

Reply:
I kept the comment to show what I was trying to mock, but I could have added some more details.

Reply:
yes more details sounds good.